apps/studio/components/interfaces/TableGridEditor/GridHeaderActions.tsx

```diff
@@ -105,6 +120,28 @@ const GridHeaderActions = ({ table }: GridHeaderActionsProps) => {
     })
   }
 
+  const closeConfirmModal = () => {
+    setRlsConfirmModalOpen(false)
+  }
+
+  const onToggleRLS = async () => {
+    const payload = {
+      id: table.id,
+      rls_enabled: !table.rls_enabled,
+    }
+
+    const res: any = await meta.tables.update(payload.id, payload)
+    if (res.error) {
+      return ui.setNotification({
```

Comment:
General comment - I think we are now using error toast messages wrongly as a default. It was meant to be a fallback rather than a consistent pattern. 

Going forward I think we should be aiming towards inline error messages that are actually in the UI you're interacting with.